mode/fortran/fortran.js

```diff
@@ -20,6 +20,10 @@ CodeMirror.defineMode("fortran", function() {
     return keys;
   }
 
+  function wordRegexp(words) {
+    return new RegExp("^((" + words.join(")|(") + "))\\b");
```

Comment:
Would this not work without the extra parentheses around every word? I didn't benchmark, but it seems that, when matching such a regexp, the regexp engine would be forced to allocate a bunch of empty match group slots in the result array, which seems wasteful.